201-diagnostics-alerts-on-premium-redis/azuredeploy.json

```diff
@@ -0,0 +1,233 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "redisCacheName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the Azure Redis Cache to create."
+      }
+    },
+    "redisCacheLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The location of the Redics Cache. For best perfomance, use the same location as the app to be used with the cache."
+      }
+    },
+    "diagnosticsStorageAccountName": {
```

Comment:
Suggestion: For any parameters that refer to existing resources, please prefix "existing" to parameter names